### PR TITLE
Fix add urlsegment

### PR DESCRIPTION
**Pull request by ericlantz**

### CODE REVIEW

**Review by wallymathieu — APPROVED** [review]
Nice!

File: src/Carable.AssemblyPayments/Implementations/ItemRepository.cs [hunk 1]
```diff
@@ -239,7 +239,7 @@ public ItemRepository(IRestClient client, ILoggerFactory loggerFactory, IOptions
         public async Task<Item> ReleasePaymentAsync(string itemId, int releaseAmount)
         {
             AssertIdNotNull(itemId);
-            var request = new RestRequest("/items/:id/release_payment", Method.PATCH);
+            var request = new RestRequest("/items/{id}/release_payment", Method.PATCH);
             request.AddUrlSegment("id", itemId);
```
Comment (wallymathieu) [inline src/Carable.AssemblyPayments/Implementations/ItemRepository.cs:243]: so the replace url segment looks for {variable} ?

**Review by mjirhage — APPROVED** [review]

File: src/Carable.AssemblyPayments/Implementations/ItemRepository.cs (hunk 1 above)
Comment (ericlantz) [reply src/Carable.AssemblyPayments/Implementations/ItemRepository.cs:243]: I have been searching around, and I can't fix a single example of them using :variable, only {varable}

Comment (wallymathieu) [reply src/Carable.AssemblyPayments/Implementations/ItemRepository.cs:243]: aa, then it's clearly a bug! Good find!

**Pull request merged**